flexmeasures/api/v1_1/tests/test_api_v1_1.py

```diff
@@ -186,7 +186,7 @@ def test_post_price_data_invalid_unit(setup_api_test_data, client, post_message)
 
 @pytest.mark.parametrize(
     "post_message",
-    [message_for_post_weather_data(), message_for_post_weather_data(temperature=True)],
+    [message_for_post_weather_data(), message_for_post_weather_data()],
```

Comment:
Thanks. I removed the test for temperature because it uses the deprecated EA addressing and tha tlooks up deprecated models (`WeatherSensor`). I had added a legacy weather sensor for this test, now I added two.